rnd/autogpt_server/setup.py

```diff
@@ -57,6 +59,26 @@ def txt_to_rtf(input_file: Union[str, Path], output_file: Union[str, Path]) -> N
 license_file = "LICENSE.rtf"
 txt_to_rtf("../../LICENSE", license_file)
 
+# call npm run build in ../autogpt_builder
+
+
+os.system("npm run build --prefix ../autogpt_builder")
+
+# copy the ../autogpt_builder/out to frontend
+
+
+shutil.rmtree("../frontend", ignore_errors=True)
+shutil.copytree("../autogpt_builder/out", "../frontend")
```

Comment:
`/out`? afaics this is currently set to `/output` (and I proposed `/dist`)

Reply:
that param doesn't do what it seems like :)

Reply:
Can you explain? I looked it up in the docs and am none the wiser.